src/Components/Artwork/GridItem.tsx

```diff
@@ -82,7 +82,10 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
     const width = 400
     const height = Math.floor(width / this.props.artwork.image.aspect_ratio)
     const type = this.props.artwork.image.aspect_ratio ? "fit" : "fill" // Either scale or crop, based on if an aspect ratio is available.
-    const url = `${sd.GEMINI_CLOUDFRONT_URL}/?resize_to=${type}&width=${width}&height=${height}&quality=${IMAGE_QUALITY}&src=${encodeURIComponent(imageURL)}` // prettier-ignore
+    const geminiUrl =
+      sd.GEMINI_CLOUDFRONT_URL || "https://d7hftxdivxxvm.cloudfront.net" // fallback, useful if we're yarn linking
```

Comment:
Instead of hardcoding this should we just pull from `sd.GEMINI_CLOUDFRONT_URL || process.env.GEMINI_CLOUDFRONT_URL`? I'm not sure which is more idiomatic but doesn't seem _great_ to have the URL in code here.